### PR TITLE
Update installed-extensions extension

### PR DESCRIPTION
**Pull request by pernielsentikaer**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by erics118** [comment]
Is jq really necessary here? Can't we directly use TypeScript handle the same parsing of JSON that jq does?

File: extensions/installed-extensions/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  }
+
+  const [installedExtensions, setInstalledExtensions] = useState<dataType[]>([]);
+  const { isLoading, data, error } = useCachedPromise(async () => {
+    const { stdout, stderr } = await exec(
+      `find ~/.config/raycast/extensions/**/package.json -exec echo -n "{}: " \\; -exec ${jqPath} -r '. | "\\(.author) \\(.icon) \\(.commands | length) \\(.name) \\(.owner) \\(.title)"' {} \\;`,
+    );
+
+    if (stderr) {
+      showFailureToast("Correct the path to jq or dowbload jq", {
```
Comment (erics118) [inline extensions/installed-extensions/src/index.tsx:38]: Typo: `dowbload` should be `download`

File: extensions/installed-extensions/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        primaryAction: {
+          title: "Download jq",
+          onAction: () => {
+            open("https://jqlang.github.io/jq/download/");
+          },
+        },
+      });
+
+      throw new Error("Correct the path to jq or dowbload jq");
```
Comment (erics118) [inline extensions/installed-extensions/src/index.tsx:47]: Typo: `dowbload` should be `download`

File: extensions/installed-extensions/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+import { useState } from "react";
+import { promisify } from "util";
+import { dataType, optionType } from "./types";
+import { extensionTypes } from "./constants";
+import { formatItem, formatOutput } from "./utils";
+
+const exec = promisify(execCb);
+
+export default function IndexCommand() {
+  const preferenes = getPreferenceValues<Preferences.Index>();
```
Comment (erics118) [inline extensions/installed-extensions/src/index.tsx:24]: Typo: `preferenes` should be `preferences`

File: extensions/installed-extensions/src/types.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export type dataType = {
```
Comment (erics118) [inline extensions/installed-extensions/src/types.ts:1]: Types are generally in camel case

File: extensions/installed-extensions/package.json [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+          "title": "Item Format",
+          "placeholder": "{title} ({link})",
+          "default": "{title} ({link})"
+        },
+        {
+          "name": "prepend",
+          "description": "The prepended text, supports {count}",
+          "type": "textfield",
+          "required": false,
+          "title": "The Prepended Text When Copying all Items",
```
Comment (erics118) [inline extensions/installed-extensions/package.json:59]: `all` should be capitalized, as part of standard title case conventions

**Comment by pernielsentikaer** [comment]
@erics118 and no, there is no reason to use `jq` besides that it was faster - might try to solve it without at some point, feel free to fix if you want 🙂

**Pull request closed without merging**